CRM/Upgrade/Incremental/php/FourSeven.php

```diff
@@ -303,6 +303,32 @@ public function upgrade_4_7_16($rev) {
     $this->addTask(ts('Upgrade DB to %1: SQL', array(1 => $rev)), 'runSql', $rev);
   }
 
+  /**
+   * Upgrade function.
+   *
+   * @param string $rev
+   */
+  public function upgrade_4_7_17($rev) {
+    $this->addTask('CRM-19385 - Look at REMOVING id column from cache tables', 'removeIdFromCacheTables');
```

Comment:
@jitendrapurohit Instead of creating a custom callback function here, I'd like to follow the same pattern as `CRM_Upgrade_Incremental_Base::addColumn` - please create a similar function like `dropColumn`. You'll need to call it 3 times from this upgrade step but that's fine.